### PR TITLE
Some door mappings

### PR DESCRIPTION
**Pull request by YanisBft**

### CODE REVIEW

File: mappings/net/minecraft/block/DoorBlock.mapping [hunk 1]
```diff
@@ -65,5 +70,7 @@ CLASS biw net/minecraft/block/DoorBlock
 	METHOD a appendProperties (Lbqj$a;)V
 		ARG 1 builder
 	METHOD c getRenderLayer ()Lbbu;
+	METHOD d getClosingSound ()I;
```
Comment (asiekierka) [inline mappings/net/minecraft/block/DoorBlock.mapping]: That's not a sound, that's an int

**Review by asiekierka — CHANGES_REQUESTED** [review]

**Review by shadowfacts — APPROVED** [review]

File: mappings/net/minecraft/block/DoorBlock.mapping [hunk 2]
```diff
@@ -70,7 +70,7 @@ CLASS biw net/minecraft/block/DoorBlock
 	METHOD a appendProperties (Lbqj$a;)V
 		ARG 1 builder
 	METHOD c getRenderLayer ()Lbbu;
-	METHOD d getClosingSound ()I;
-	METHOD e getOpeningSound ()I;
+	METHOD d getClosingSound ()Lxr;
+	METHOD e getOpeningSound ()Lxr;
```
Comment (modmuss50) [inline mappings/net/minecraft/block/DoorBlock.mapping:74]: Are these methods actaully returning an int or `xr`?

Can these mappings be checked to see if they are still valid.

**Review by modmuss50 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/block/DoorBlock.mapping (hunk 2 above)
Comment (Chocohead) [reply mappings/net/minecraft/block/DoorBlock.mapping:74]: They get passed into `World#fireWorldEvent` so should be `()I`

Comment (YanisBft) [reply mappings/net/minecraft/block/DoorBlock.mapping:74]: Oh my bad I didn't get what asie requested first

Comment (Juuxel) [reply mappings/net/minecraft/block/DoorBlock.mapping:74]: `getClosingWorldEvent`?

**Pull request closed without merging**